### ISSUE TITLE
[Feature Request] Message Filter - Configrurable minutes for periodical filter run

### ISSUE THREAD

**Issue by hermit99** [Betterbird/thunderbird-patches#268]
The current filter trigger of periodical run is hard coded to **10** minutes. I would like to see a feature / UI enhancement that allows users to configure this **X** minutes, so that the message filter can be used as an archiving / folder cleaning tool on a typically much lower frequency than every 10 minutes.

![image](https://github.com/Betterbird/thunderbird-patches/assets/70563349/ef9460eb-56ac-4ee1-bed1-81a8206426a9)

**Comment by Betterbird** [comment]
Would you like to sponsor?

**Comment by hermit99** [comment]
Not sure how much is your funding requirement, but happy to donate.

**Comment by Betterbird** [comment]
This code was added in https://bugzilla.mozilla.org/show_bug.cgi?id=864187. The time is actually flexible:
https://hg.mozilla.org/comm-central/rev/686c2fa1e7cb#l10.13
So set pref mail.server.default.periodicFilterRateMinutes to whatever you want.
Or check "Help > Troubleshooting Information" for the server you want to change and set mail.server.serverNN.periodicFilterRateMinutes.

Here's the result when I set mail.server.server1.periodicFilterRateMinutes.
![image](https://github.com/Betterbird/thunderbird-patches/assets/86832723/55138d08-80e6-44aa-a2ba-b14f59c2d885)

I think we won't invest any further work here.
